modules/snooker/src/main/ts/ephox/snooker/api/Generators.ts

```diff
@@ -1,21 +1,25 @@
-import { Arr, Fun, Optional, Optionals, Singleton } from '@ephox/katamari';
+import { Arr, Fun, Optional, Optionals } from '@ephox/katamari';
 import { Attribute, Css, SugarElement, SugarNode } from '@ephox/sugar';
 
 import { getAttrValue } from '../util/CellUtils';
 
-export interface CellSpan {
+export interface RowData {
+  readonly element: SugarElement<HTMLTableRowElement | HTMLTableColElement>;
+}
+
+export interface CellData {
```

Comment:
Should this rename get a changelog entry?

Reply:
It's not publicly exposed outside snooker which is why I didn't include it.

Reply:
Oh I see.

Out of scope, but is it a problem that something exported from `/api/*` isn't then exported from the `/api/Main`?

Reply:
I think you've misunderstood... `Generators` is exported, but only parts of it. Here's what's exported:
```
import { Generators, SimpleGenerators } from './Generators';
```

Reply:
Yeah no that's what I mean. My understanding was that normally for things like that we'd have `api/Generators` **only** export those two interfaces, and make a `core/Generators` or similar that has the rest of those interfaces (maybe even putting `Generators` and `SimpleGenerators` into that file and re-exporting them from api).

To be super clear, I'm not at all asking for a change here, just trying to clarify our own code style

Reply:
That understanding is incorrect. Anything in `api` just means that something from that module is used publicly, not everything.